matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/membership/RoomMemberHelper.kt

```diff
@@ -98,6 +98,10 @@ internal class RoomMemberHelper(private val realm: Realm,
         return getNumberOfJoinedMembers() + getNumberOfInvitedMembers()
     }
 
+    fun isDirectRoom() : Boolean {
```

Comment:
This looks a bit offtopic regarding the name `RoomMemberHelper`. Isn't an other place to put this method?

Reply:
The thing is that i also need the isDM info as well as room member info, and the helper already did the query, seems annoying to redo the same query as at could already get this info from the helper

Reply:
done